### PR TITLE
Add Bodhi Linux

### PR DESCRIPTION
**Pull request by zen0bit**

### CODE REVIEW

**Comment by zen0bit** [comment]
running vm not yet tested

File: quickget [hunk 1]
```diff
@@ -293,6 +294,14 @@ function editions_blendos() {
     echo gnome kde
 }
 
+function releases_bodhi() {
+    echo 7.0.0
+}
+
+function editions_bodhi() {
+    echo standart hwe s76
```
Comment (philclifford) [inline quickget:302]: 's/ standart / standard /g'

File: quickget [hunk 2]
```diff
@@ -1103,6 +1112,21 @@ function get_batocera() {
...
+    local EDITION="${1:-}"
+    local HASH=""
+    local ISO=""
+    local URL="https://sourceforge.net/projects/bodhilinux/files/${release}/"
+
+    case ${EDITION} in
+      standard) ISO="bodhi-${RELEASE}-64.iso";;
+      hwe) ISO="bodhi-${RELEASE}-64-hwe.iso";;
+      s76) ISO="bodhi-${RELEASE}-64-s76.iso";;
+    esac
```
Comment (philclifford) [inline quickget:1125]:
```suggestion
      *) ISO="bodhi-${RELEASE}-64-${EDITION}.iso";;
    esac
```

**Review by philclifford — APPROVED** [review]

**Comment by zen0bit** [comment]
bodhi

**Pull request merged**